### ISSUE TITLE
Attention layer does not support masking. 

### ISSUE THREAD

**Issue by sagar-spkt** [thushv89/attention_keras#8]
Any ideas about implementing it??

**Comment by thushv89** [comment]
Hi @sagar-spkt ,
Can you elaborate on what sort of masking you are referring to?

**Comment by adrian-dalessandro** [comment]
Hey @thushv89,

there problem that @sagar-spkt is referring to is masking with respect to LSTM sequence length. It's the way Keras lets you control the input of variable length sequences.

You can read about it here: https://www.tensorflow.org/guide/keras/masking_and_padding#supporting_masking_in_your_custom_layers

It's actually not too difficult to implement at the layer level.

**Comment by thushv89** [comment]
@adrian-dalessandro 

Thanks. I'll have a look into this this week. I was busy with some other work and really didn't have time to look into this repo.
